content/en/docs/tasks/manage-kubernetes-objects/declarative-config.md

```diff
@@ -83,7 +83,14 @@ kubectl diff -f https://k8s.io/examples/application/simple_deployment.yaml
 ```
 
 {{< note >}}
-`diff` uses [server-side dry-run](/docs/reference/using-api/api-concepts/#dry-run), which needs to be enabled on `kube-apiserver`.
+`diff` uses [server-side dry-run](/docs/reference/using-api/api-concepts/#dry-run),
+which needs to be enabled on `kube-apiserver`.
+
+Since dry-run requires write authorization to PATCH,
```

Comment:
suggestion:

Since diff performs a server-side apply request in dry-run mode, it requires granting `patch`, `create`, and `update` permissions. See [Dry Run Authorization](/docs/reference/using-api/api-concepts#dry-run-authorization) for details.

Reply:
Thanks for this suggestion, this wording is clearer. Updated